.CI/create_feedstocks

```diff
@@ -9,6 +9,7 @@ if [ -n "$GH_TOKEN" ]; then
     conda config --set show_channel_urls true
     conda config --add channels conda-forge
     conda install --yes --quiet conda-smithy
+    conda install --yes --quiet conda-build=1.21.11
```

Comment:
Added a comment to explain this in PR ( https://github.com/conda-forge/staged-recipes/pull/1299 ).